api/pkg/apis/v1alpha1/vendors/catalogs-vendor.go

```diff
@@ -222,6 +232,11 @@ func (e *CatalogsVendor) onCatalogs(request v1alpha2.COARequest) v1alpha2.COARes
 	defer span.End()
 
 	lLog.Info("V (Catalogs Vendor): onCatalogs")
+	scope, exist := request.Parameters["scope"]
```

Comment:
I think we only need to respect scope parameter in request in catalogs-vendor (this is the right place). It means we want to create/get/list/delete catalog in certain namespace. 
You need to review other vendors that you check scope parameter. I don't think there are needed in those scenarios.

Reply:
Didn't get your point. This is what we're doing - respecting scope parameter in request in catalogs-vendor.